src/bin/gpfusion/gpbridgeapi.c

```diff
@@ -536,3 +536,8 @@ void free_token_resources(PxfInputData *inputData)
 
 	pfree(inputData->token);
 }
+
+void free_dfs_address()
+{
+	free(dfs_address);
+}
```

Comment:
Oh right, good catch. I will add a check in `free_dfs_address` for secure filesystem. Sound good?